app/components/support_interface/application_add_course_component.html.erb

```diff
@@ -2,9 +2,9 @@
   <p class='govuk-body'>
     This candidate can add <%= application_form.maximum_number_of_course_choices %> course choices to this application.
   </p>
-<% elsif application_form.has_the_maximum_number_of_course_choices? %>
+<% elsif application_form.support_cannot_add_course_choice? %>
   <p class='govuk-body'>
-    This application already has the maximum number of course choices (<%= application_form.maximum_number_of_course_choices %>), so we can't add any more.
+    This application already has the maximum number of active course choices (<%= application_form.maximum_number_of_course_choices %>), so we can't add any more.
```

Comment:
Is it worth adding a note about requesting candidate to withdraw one of the three options if required, in order for this feature to become available; to avoid any relevant support requests

Reply:
That's a good idea I think. In the support doc we give instructions to withdraw applications as well so maybe there's a case for making 'Withdraw application' a feature of the support interface too. What do you think?

Reply:
I think it's better to ask candidates to do that, as this option is already available to them (and the support agent would have to double check and confirm withdrawing any of their available options anyway as well as explain why they are doing it).